raiden-dapp/src/components/ConfirmationDialog.vue

```diff
@@ -1,95 +1,67 @@
 <template>
-  <v-row class="confirmation" no-gutters>
-    <v-row align="center" justify="center">
-      <h1 class="confirmation__text__header"><slot name="header"></slot></h1>
-    </v-row>
-    <v-row align="center" justify="center">
-      <v-col cols="12">
-        <p class="confirmation__text__message"><slot></slot></p>
-      </v-col>
-    </v-row>
-    <v-row
-      align="end"
-      justify="center"
-      class="confirmation__buttons"
-      no-gutters
-    >
-      <v-btn
-        :id="`cancel-${identifier}`"
-        class="text-capitalize confirmation__buttons__cancel"
-        @click="cancel()"
-      >
-        {{ $t('confirmation.buttons.cancel') }}
-      </v-btn>
-      <v-btn
-        :id="`confirm-${identifier}`"
-        class="text-capitalize confirmation__buttons__confirm"
-        @click="confirm()"
-      >
-        {{ positiveAction }}
-      </v-btn>
-    </v-row>
-  </v-row>
+  <raiden-dialog
+    class="confirmation"
+    :visible="visible"
+    :fullscreen="false"
+    @close="cancel"
+  >
+    <v-card-title>
+      <v-row align="center" justify="center">
+        <v-col>
+          <span>
+            <slot name="header"></slot>
+          </span>
+        </v-col>
+      </v-row>
+    </v-card-title>
+
+    <v-card-text>
+      <v-row align="center" justify="center">
+        <span>
+          <slot></slot>
+        </span>
+      </v-row>
+    </v-card-text>
+
+    <v-card-actions>
+      <v-row align="center" justify="center">
```

Comment:
Does it need the `v-row` and `v-col` here?

Reply:
Apparently not, it seems to work perfectly fine without it. :) I should look over the other dialogs as well then and make sure I'm not using redundant `v-row` and `v-col`.

Reply:
The other ones seem to be giving me some weird behavior in the layout so I will leave them as-is for now and only change the `ConfirmationDialog`.